interface/src/app/gene/searchFew/searchFew.js

```diff
@@ -51,17 +50,21 @@ angular.module('adage.gene.searchFew', [
               } else {
                 $scope.noResults = false;
               }
-
-              return data.results;
             },
 
             // Error function
             function(responseObject, responseHeaders) {
               $scope.errors = 'Gene search is temporarily down';
             }
-          ).$promise.then(function(response) {
-            return response.results;
-          });
+          ).$promise.then(
+            function(response) {
+              var selectedIDs = Object.keys($scope.selectedGenes);
+              var geneResultsList = response.results.filter(function(result) {
+                return selectedIDs.indexOf(result.id.toString()) === -1;
+              });
+              return geneResultsList.slice(0, numResultsToReturn);
+            }
+          );
```

Comment:
This is a nice simple fix that improves #69.

It's a nice catch, also, that you want to let new entries into the autocomplete list as those that are added get filtered out here. I think there's a flaw in this approach, however: you'll never be able to add more than the top `GENES_API_RESULT_LIMIT` search results.

Here's how I tested:

1. perform an autocomplete search using `PA`
2. add the top entry
3. repeat 1 & 2 fifteen times
4. typing `PA` into the autocomplete search returns no more results, even though there are still more to be found (confirmed by typing `PA1` and getting more results)

I see two ways around this.

A rigorous fix would include sending the list of genes filtered at the front end into the autocomplete API endpoint and have the back-end filter those out before building a new list of `GENES_API_RESULT_LIMIT` results to send back to the front end. This would be a complete fix but requires more updates to the whole stack (including another revision to the `django-genes` API).

A more pragmatic approach would be to specify a much higher `GENES_API_RESULT_LIMIT` (maybe 100?) and recognize that most users probably won't be using the autocomplete approach to enter more than a few dozen genes.